server.js

```diff
@@ -11,4 +11,48 @@
 // through the list, building up a string of food truck names. Once you're done 
 // iterating through that list, display the string you built up.
 
-// Remember that the response is a stream object that must be closed.
\ No newline at end of file
+// Remember that the response is a stream object that must be closed.
+
+//files: cd emma\cv\study\washington.edu\JavaScript3\Module 2\DateUtility\Module_3
+// node server.js 
+//http://127.0.0.1:3000/
+
+//building a simple http server in Node (Node.pdf)
+//- server.js
+
+var http = require('http');
+
+//trucks
+var trucks = require('./trucks');
+
+//from my date file
+var date = require('./enhanced-date');
+date.setDate();
+
+//server request and response
+http.createServer(function (request, response) {
+
+    //the output
+    var responseOutPut = 'Today is ' + date.getDayName() + ', '
+          + date.getDate('formatted') + '.' + '\n' + 'The food trucks available are:\n\n';
+
+    //trucks are open
+    var truckArray = trucks.filterByDay(date.getDayName());
+
+    //loop adding trucks
+    for (var i = 0; i < truckArray.length; i++) {
+        responseOutPut += '-' + truckArray[i].name + '\n';

```

Comment:
Your truckArray is just an array of names, not an array of objects with name properties. By trying to access a property that doesn't exist, it results in your code outputting undefined for each of the trucks.